themes/StarterTheme/theme.yml

```diff
@@ -0,0 +1,88 @@
+meta:
+  name: StarterTheme
+  display_name: Starter Theme
+  version: 1.0.0
+  author:
+    name: "PrestaShop John Doe"
+    email: "pub@prestashop.com"
+    url: "http://www.prestashop.com"
+  compatibility:
+      from: 1.7.0.0
+      to: ~
+  available_layouts:
+    # Users will be able to choose each page's layout
+    # from the theme's settings page.
+    # Layouts are automatically parsed from the templates/layouts
+    # folder so this configuration key is optional,
+    # but it allows designers to provide some more user friendly
+    # info than just a filename.
+    layout-full-width:
+      name: Full width layout
+      description: Ideal for product pages to maximize picture size
+    layout-left-side-column:
```

Comment:
About layouts, wouldn't be great to illustrate them with an image (facultative) ?